roles/baseline/defaults/main.yml

```diff
@@ -121,6 +121,23 @@ CLUSTER_AUTOSCALER_CONFIG:
       annotations:
         "eks.amazonaws.com/role-arn": "{{ CLUSTER_AUTOSCALER_ACCOUNT }}"
 
+## EBS CSI Driver
+EBS_CSI_DRIVER_ENABLED: true
```

Comment:
Not needed

Reply:
Doesn't this option, that will default to true, provide the only easy way for a DAC customer deploying to k8s_version 1.23 or later to prevent ebs csi driver installation if they were to need that for some reason? I don't see how it can be a liability if for only that reason. I added it to provide similar behavior to the autoscaling which is always installed but provides that var as a way to prevent it if needed. Just wanted to bring that up for discussion.

Reply:
As we talked the EBS driver is ALWAYS needed to provide a working `default` storage class. The objective here it to create a functioning cluster out of the box. If someone sets the flag to false we now have to walk them through setting up the driver, etc. themselves. In the case of autoscaling if that flag is set to `false` the cluster still works. Not the case with the EBS driver. If you want to hide the flag, i.e. remove it from the CONFIG_VARS.md file. I am fine with that. Then it's there not documented and we can do some testing if needed. Again, it's more a distraction at this point. AWS says you need the driver for k8s 1.23 or things won't work, so we provide it.

Reply:
I will remove it from CONFIG_VARS.md and can see the merits of doing that based on those reasons.